### ISSUE TITLE
Linux machine and android phone can't see each other

### ISSUE THREAD

**Issue by marcoppola** [GSConnect/gnome-shell-extension-gsconnect#1758]
### Describe the bug

oracle linux machine (9.3 kernel version,  gnome version 40.10) don't see android phone
android phone (version 13) with kde connect installed and running don't see laptop with Oracle linux

### Steps to reproduce

install kde connect
install gs connect
restart laptop, restart phone
try to connect

I also tried to connect the devices with ip address (on both devices) with the respective ip without success

### GSConnect version

47 (user)

### Installed from

GNOME Extensions website

### GNOME Shell version

40.10

### Linux distribution/release

Oracle Linux Server 9.3

### Paired device(s)

samsung galaxy A52

### KDE Connect app version

1.29.0

### Support log

```plain text
support log are too long and this form can't be submitted if I past it.
felle free to ask it throught another way
```

**Comment by ankit-smart-swami** [comment]
Same issue I am facing.

**Comment by NyllRE** [comment]
same issue, this should be looked into they're absolutely not finding each other
Nobara (Fedora fork) Linux 39

**Comment by rohmishra** [comment]
can you please attach the log as a text file or as a comment please?

**Comment by Aybee5** [comment]
Mine is connecting, but somethings my phone can't see my computer.

I am guessing it's because the process is somehow killed on my computer and I am wondering if there's a way to restart the gsconnect on my computer

**Comment by marcoppola** [comment]
when I opened the ticket I checked the logs and them was very long (several thousand of lines). Now only 15 lines...


`GSConnect: 47 (user)
GJS:        #16806
Session:   wayland
OS:        Oracle Linux Server 9.3
--------------------------------------------------------------------------------
Mar 25 16:43:12 google-chrome.desktop[3995]: [4044:4056:0325/164312.187844:ERROR:ssl_client_socket_impl.cc(970)] handshake failed; returned -1, SSL error code 1, net_error -202
Mar 25 16:43:16 org.gnome.Shell.Extensions.GSConnect[3449]: [/service/backends/lan.js:broadcast:413]: "Broadcasting to LAN"
Mar 25 16:43:17 systemd[2766]: Started dbus-:1.2-org.gnome.gedit@1.service.
Mar 25 16:43:39 kernel: Lockdown: systemd-logind: hibernation is restricted; see man kernel_lockdown.7
Mar 25 16:43:43 gnome-shell[2970]: JS WARNING: [resource:///org/gnome/gjs/modules/core/overrides/Gio.js 415]: Too many arguments to method Gio.DBusConnection.call: expected 10, got 11
Mar 25 16:43:43 systemd[2766]: dbus-:1.2-org.gnome.Shell.Extensions.GSConnect@0.service: Consumed 1.433s CPU time.
Mar 25 16:43:44 kernel: Lockdown: systemd-logind: hibernation is restricted; see man kernel_lockdown.7
Mar 25 16:43:46 systemd[2766]: Started dbus-:1.2-org.gnome.Shell.Extensions.GSConnect@1.service.`

**Comment by zetaomegagon** [comment]
@marcoppola your logs only have one line that I know is 100% relevant:
```
Mar 25 16:43:16 org.gnome.Shell.Extensions.GSConnect[3449]: [/service/backends/lan.js:broadcast:413]: "Broadcasting to LAN"
```

Maybe this one too, but I'm not sure if GSConnect uses SSL or something else
```
Mar 25 16:43:12 google-chrome.desktop[3995]: [4044:4056:0325/164312.187844:ERROR:ssl_client_socket_impl.cc(970)] handshake failed; returned -1, SSL error code 1, net_error -202
```

Need more logs.

For large logs you can use a paste bin service. There are many to choose from on the internet.

Please note that I'm not affiliated with this project in any way, I only sound like I know what I'm talking about.

**Comment by zetaomegagon** [comment]
@marcoppola , additionally, it would be good to know if you've read thorough this section of the wiki, and tried any of the troubleshooting steps:

https://github.com/GSConnect/gnome-shell-extension-gsconnect/wiki/Help#connecting-an-android-device

One thing that I can personally say is that if you are using any kind of VPN on your phone, it can block communication between devices. Also, IIRC, the phone passively receives a request to connect, the PC is the initiator and requests happen as soon as the extension is enabled.

**Comment by Aybee5** [comment]
> @marcoppola , additionally, it would be good to know if you've read thorough this section of the wiki, and tried any of the troubleshooting steps:
> 
> [Wiki: Help (connecting an android device) ()](https://github.com/GSConnect/gnome-shell-extension-gsconnect/wiki/Help#connecting-an-android-device)
> 
> One thing that I can personally say is that if you are using any kind of VPN on your phone, it can block communication between devices. Also, IIRC, the phone passively receives a request to connect, the PC is the initiator and requests happen as soon as the extension is enabled.

I am using VPN on my computer(Ubuntu 22.04) and I noticed it disconnect when the VPN is on, is there any workaround for this?

**Comment by Aybee5** [comment]
> I am using VPN on my computer(Ubuntu 22.04) and I noticed it disconnect when the VPN is on, is there any workaround for this?

For nordvpn allow lan-discovery

`nordvpn set lan-discovery enabled`

https://github.com/GSConnect/gnome-shell-extension-gsconnect/issues/1550#issuecomment-2113399325

**Comment by marcoppola** [comment]
@zetaomegagon: yes, I tried the steps on the wiki and both the devices don't see each other. Also adding the device with the ip address doesn't work.

I don't use any VPN.

I tried to generate another time the logs, but the file generated is empty.

**Comment by zjcz** [comment]
Have you configured the firewall to allow GSConnect through?  I had a similar issue when installing on Ubuntu 24.04 and I had to configure ufw to open the ports.

https://github.com/GSConnect/gnome-shell-extension-gsconnect/wiki/Help#firewall-configuration

**Comment by zjcz** [comment]
I can't find anything concrete, but a quick internet search seems to imply that ports 1026 and above are open by default in Fedora. ([Fedora Discussion 1](https://discussion.fedoraproject.org/t/all-non-well-known-ports-open-in-firewalld-open-by-default/60177), [Fedora Discussion 2](https://discussion.fedoraproject.org/t/firewall-default-settings-config-after-fresh-install-fedora-37-ws/80568/2), [Fedora Discussion 3](https://discussion.fedoraproject.org/t/how-to-open-a-tcp-port-using-firewall-config/73818/10)).  

Maybe this has changed in Fedora 40 or something else has locked it down for you?  I'm not a Fedora user so can't help I'm afraid
